api/src/documents.py

```diff
@@ -507,30 +505,6 @@ def get_document_text(document_id: str,
     }
 
 
-@router.get("/documents/by-didx-name", include_in_schema=False)
-def get_document_by_didx_name(name: str):
```

Comment:
This was only used for our original knowledge ai demo- since the reference PDFs were returned by **filename** from boxytas local dir, and we needed a way to match dojo's `document_id` to the boxytas hits.

The boxytas knowledge ai server code was ported to dojo, and now everything uses elasticsearch (no local filesystem files to match filenames to).

Reply:
@adamgilg I think you were aware of this workaround. Just noting it here for future reference. Removing as part of the consolidate-llm work.